src/rinq/attr.go

```diff
@@ -43,15 +46,15 @@ func (attr Attr) String() string {
 		} else {
 			buf.WriteString("-")
 		}
-		buf.WriteString(attr.Key)
+		buf.WriteString(repr.Escape(attr.Key))
```

Comment:
Could you please add a new test that verifies `Attr.String()` actually does the escaping.  Using keys/values that contain a space should be adequate. Tests are in `attr_test.go` and use the Ginkgo/Gomega frameworks.